src/graphnet/deployment/i3modules/graphnet_module.py

```diff
@@ -153,42 +115,299 @@ def _extract_feature_array_from_frame(self, frame: I3Frame) -> np.array:
                 )
         return features
 
-    def _write_predictions_to_frame(
-        self, frame: I3Frame, prediction: np.array
+    def _submit_to_frame(
```

Comment:
I figure "add" might me more intuitive?
```suggestion
    def _add_to_frame(
```